payments/index.md

```diff
@@ -1,8 +1,209 @@
+  An authorization transaction reserves the funds necessary funds . It is possible to abort a payment before the end user has fulfilled the payment process.  
+And either:  
++ Capture funds.  
+  Before delivering the merchandise you need to create a capture transaction to ensure that the money is charged from the consumer credit card or properly billed by invoice.  
+Or:  
++ Cancel the authorized amount.  
+ Funds that are authorized but not yet captured can be released back to the consumer. This is done by creating a cancel transaction. 
++ Reverse captured funds.  
+ In some cases you may need to make a reversal of captured funds. This is achieved by creating a reversal transaction.
+
+
+*One-phase payments* ([Direct Debit](/payments/direct-debit), [Swish](/payments/swish), [Card](/payments/credit-card))
+
+If you use a one-phase method (like Direct Bank Debit or [Swish](/payments/swish)) a Sales transaction will be created and the consumer charged right away. 
+
+* _One-phase card payments is enabled with autocapture._
+
+>Add image
+
+### <a name="creating-a-payment-object">Creating a payment object</a>.
+The `payment` is the container object that holds all transactions that will be created during the payment process. When PayEx receives the payment request body (in JSON format), a payment  is created and you will be given a unique paymentID in return. The response also include  (in a true RESTful way) the URIs to possible further actions, given the state of the payment. It is possible to abort a payment before the end user has fulfilled the payment process.
+Settle funds directly with a Sales transaction.  
+```
+Pick and choose between the payment methods best suited for your business. Take advantage of our easy-to-use PCI compliant platforms Redirect and PayEx Hosted View - or use PayEx Direct API to integrate directly. Our payment methods and their platform availability are listed in the table below.
+```
+
+{:.table .table-striped}
+|  | Payment method | Hosted View | Redirect | Direct API | Region |
+| CardIcon | [Credit card](payment/credit-card/) | ✔︎︎︎︎︎ | ✔︎︎︎︎︎ | ✔︎︎︎︎︎ | EarthIcon |
+| InvoiceIcon | [PayEx Invoice](payment/invoice/) | ✔︎︎︎︎︎ | ✔︎︎︎︎︎ | ✔︎︎︎︎︎ | NordicCountries |
+| EnvelopeIcon | [PayEx Web Invoice](payment/credit-account) | ✔︎︎︎︎︎ | ✔︎︎︎︎︎ | ✔︎ ︎| SwedenNorway|
+| KeypadIcon | [Direct Debit](payment/direct-debit)| X | ✔︎︎︎︎︎ | X | BalticNordic |
+| Vipps | [Vipps](payment/vipps) | ✔︎︎︎︎︎ | ✔︎︎︎︎︎ | X | Norway |
+| Swish | [Swish](payments/swish) | ✔︎︎︎︎︎ | ✔︎︎︎︎︎ | ✔︎︎︎︎︎  ︎| Sweden |
+| MobilePay | [Mobile Pay](payments/mobile-pay) | X | ✔︎︎︎︎︎ | X | Denmark |
+
+>Add image
+
+## PayEx Payment Instruments Platforms
+PayEx Hosted view and Redirect offer easy-to-use PCI compliant platforms, available from both web and mobile browsers. Either let your customers access the platform directly, embedded on your site, or by redirection to a separate hosted payment page.
+
+### Hosted View implementation
```

Comment:
I've realized that we need to create one section per integration variant:

1. Redirect
2. Seamless View (aka Hosted View)
3. Tailored (for Checkout)

Each section needs to describe its integration variant fully, from A to Z.

Reply:
Okay, should we do that now or later?

Reply:
Perhaps we can do it in a separate task and PR. It might involve a lot of work.